### PR TITLE
Bump the ansible-lint version to 24.7.0

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

Bump the ansible-lint version to 24.7.0

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: .github/workflows/linters.yaml [hunk 1]
```diff
@@ -21,5 +21,5 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Run ansible-lint
-        uses: ansible/ansible-lint@v6.21.0
\ No newline at end of file
+      - name: run-ansible-lint
```
Comment (alinabuzachis) [inline .github/workflows/linters.yaml:24]: Why the change of step name?

**Review by alinabuzachis — APPROVED** [review]

File: .github/workflows/linters.yaml (hunk 1 above)
Comment (mandar242) [reply .github/workflows/linters.yaml:24]: Replaced it with new task from https://github.com/marketplace/actions/run-ansible-lint

Comment (mandar242) [reply .github/workflows/linters.yaml:24]: image

**Review by gravesm — APPROVED** [review]
